community/zstd/APKBUILD

```diff
@@ -1,17 +1,16 @@
 # Contributor: stef <l0ls0fo2i@ctrlc.hu>
 # Maintainer: André Klitzing <aklitzing@gmail.com>
 pkgname=zstd
-pkgver=1.3.5
+pkgver=1.3.7
 pkgrel=0
 pkgdesc="Zstandard - Fast real-time compression algorithm"
 url="http://www.zstd.net"
 arch="all !armhf !armv7"
 license="BSD-3-Clause GPL-2.0-or-later"
 checkdepends="file"
+makedepends="grep"
 subpackages="$pkgname-static $pkgname-libs $pkgname-dev $pkgname-doc"
-source="$pkgname-$pkgver.tar.gz::https://github.com/facebook/$pkgname/archive/v$pkgver.tar.gz
-	allow-invoking-list-when-stdin-not-tty.patch
-	fix-list-tests-with-stdin.patch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/facebook/$pkgname/archive/v$pkgver.tar.gz"
```

Comment:
While you're on it, please replace `s/https://github.com/facebook/$pkgname/https://github.com/facebook/zstd` to easy navigation